Resources/translations/content_type.en.xlf

```diff
@@ -1,149 +1,231 @@
 <?xml version="1.0" encoding="utf-8"?>
-<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
-  <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
+  <file date="2016-11-29T11:49:06Z" source-language="en" target-language="en" datatype="plaintext" original="not.available">
     <header>
-      <tool tool-id="symfony" tool-name="Symfony"/>
+      <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
+      <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
     </header>
     <body>
-      <trans-unit id="4f1cfd17c4ea63fdcfb4a10fda0755ff" resname="content_type.dashboard_title">
-        <source>content_type.dashboard_title</source>
-        <target>Content types</target>
+      <trans-unit id="d6c5835e7dba84e22f3f6cfd985dbff2ba5f7494" resname="content_type.always_available">
+        <source>Available</source>
+        <target>Available</target>
+        <note>key: content_type.always_available</note>
+        <jms:reference-file line="73">/../../../../.././Resources/views/ContentType/view_content_type.html.twig</jms:reference-file>
```

Comment:
Yeah, I know. The different extractors don't behave the same way, and it produces this. It's either a lot of dots, or `vendor/...` :)

I could maybe handle that in the custom file-writer, but it does qualify as a bugfix, does it not ? ;)